app/Template/board/task_footer.php

```diff
@@ -65,7 +65,7 @@
                 <?php endif ?>
                 ">
                 <i class="fa fa-calendar"></i>
-                <?= $this->dt->date($task['date_due']) ?>
+                <?= $this->dt->date($task['date_due']) ?> <?= $this->dt->time($task['date_due']) ?>
```

Comment:
Instead of using the `time()` function, can you use directly the helper `$this->dt->datetime($task['date_due'])`?